bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaGatewayHandler.java

```diff
@@ -39,21 +40,20 @@ public SomfyTahomaGatewayHandler(Thing thing) {
 
     @Override
     public void initialize() {
-        if (getBridge().getStatus() == ThingStatus.ONLINE) {
```

Comment:
I think you don't need to override if you do nothing but calling super.

Reply:
OK